build-docker-images/wazuh-manager/Dockerfile

```diff
@@ -1,27 +1,22 @@
 # Wazuh Docker Copyright (C) 2021 Wazuh Inc. (License GPLv2)
-FROM centos:7
+FROM ubuntu:focal
 
+ARG WAZUH_VERSION=4.3.0
+ARG TEMPLATE_VERSION="master"
```

Comment:
Why master? This could be dangerous if we start working on Elastic Common Schema and we modify the template.

Reply:
It is a variable that was assigned that way in the previous version, I try to eliminate it and see the deploy behavior

Reply:
the variable was modified by the value of the branch used in the version